### ISSUE TITLE
Add screenreader help for active polls

### ISSUE THREAD

**Issue by Elblinator** [OpenSlides/openslides-client#6138]
**Describe the bug**
in the following PR the assertiveness level of polls was changed: https://github.com/OpenSlides/openslides-client/pull/5980

But we were told that the assertiveness is not working correctly. 
Currently if a poll is started, that is not indicated while using the software.
The examples where not from the autopilot but from the respective detail views (motion, topic and election)

While in the motion, if in that motion a poll is started nothing about hte poll is read out loud

**How to Reproduce**
1. use NVDA screenreader
2. be in a motion
3. have active focus in that motion
4. on a different device start a poll in that motion
5. the title from the poll is not read out loud

**Expected behavior**
The poll title should be read out loud 
OR any indication that that poll is now there and started should be read out loud
